src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -68,7 +68,8 @@ export const fetchDictionaries = () => (dispatch) => {
       dispatch(isFetching(false));
     })
     .catch((error) => {
-      dispatch(isErrored(error.response.data));
+      error.response ? dispatch(isErrored(error.response.data)):
+        notify.show("Network Error. Please try again later!", 'error', 6000);
```

Comment:
Can you avoid duplicating the above error message?

Reply:
This has been fixed.